libs/UIHelpers/stb_image_resize.h

```diff
@@ -0,0 +1,2634 @@
+/* stb_image_resize - v0.97 - public domain image resizing
+   by Jorge L Rodriguez (@VinoBS) - 2014
+   http://github.com/nothings/stb
```

Comment:
Is this an external library you've added? Do you need all the comments at the top of the file?